src/styles/info-label.scss

```diff
@@ -3,31 +3,34 @@
 
 $block: #{$fd-namespace}-info-label;
 
-$fd-info-label-height: 1.125rem !default;
+$fd-info-label-height: 1rem !default;
 $fd-info-label-display-height: 1rem !default;
-$fd-info-label-icon-text-spacing: 0.1875rem !default;
-$fd-info-label-border-width: 0.063rem !default;
+$fd-info-label-icon-text-spacing: 0.125rem !default;
```

Comment:
if `$fd-info-label-height` is now 1rem maybe remove `$fd-info-label-display-height` as they are now the same

Reply:
removed